<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_ssh">
      <attribute name="title">
       <string>SSH</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_use_separete_known_hosts">
         <property name="text">
          <string>Set separate &amp;known hosts</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QTextEdit" name="textEdit_known_hosts"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBox_use_internal_ssh_library">
         <property name="text">
          <string>Use &amp;internal OpenSSH library</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_ssh_program">
         <property name="text">
          <string>SSH program path</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="lineEdit_ssh_program"/>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="browse_ssh_program_button">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_ssh_args">
         <property name="text">
          <string>Addional options for SSH</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="lineEdit_ssh_args"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_environment">
         <property name="text">
          <string>Addional Environment:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTreeWidget" name="environment_list">
           <property name="editTriggers">
            <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
           </property>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="button_add_env_item">
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_remove_env_item">
             <property name="maximumSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_chat">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_font">
         <property name="text">
          <string>Default font family</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Default font size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_font_size">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_use_separete_known_hosts</sender>
   <signal>toggled(bool)</signal>
   <receiver>textEdit_known_hosts</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_use_internal_ssh_library</sender>
   <signal>toggled(bool)</signal>
   <receiver>SettingsDialog</receiver>
   <slot>use_internal_ssh_library_checked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_add_env_item</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>add_environment_variable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_remove_env_item</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>remove_environment_variable()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>361</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse_ssh_program_button</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>browse_ssh_program()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SettingsDialog</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>save_settings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>use_internal_ssh_library_checked(bool)</slot>
  <slot>add_environment_variable()</slot>
  <slot>remove_environment_variable()</slot>
  <slot>browse_ssh_program()</slot>
  <slot>save_settings()</slot>
 </slots>
</ui>
